src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/repo.py

```diff
@@ -23,8 +22,6 @@
     DEPLOYMENT_ACCOUNT_REGION,
     S3_BUCKET_NAME
 )
-
-
 class Repo:
```

Comment:
The lines that were removed before the class should be there.
Adding these as a suggestion here, as GitHub doesn't support suggestions on deleted lines (odd).
```suggestion


class Repo:
```